error compiling (CV_CAP_PROP_FRAME_WIDTH)

### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Compile latest git on Arch Linux
```
yay -G performous-git
cd performous-git
makepkg
..
[ 93%] Building CXX object game/CMakeFiles/performous.dir/video_driver.cc.o
[ 94%] Building CXX object game/CMakeFiles/performous.dir/webcam.cc.o
/home/jose/performous-git/src/performous-git/game/webcam.cc: In constructor 'Webcam::Webcam(int)':
/home/jose/performous-git/src/performous-git/game/webcam.cc:36:21: error: 'CV_CAP_PROP_FRAME_WIDTH' was not declared in this scope
  if (m_capture->get(CV_CAP_PROP_FRAME_WIDTH) < 640
                     ^~~~~~~~~~~~~~~~~~~~~~~
/home/jose/performous-git/src/performous-git/game/webcam.cc:37:22: error: 'CV_CAP_PROP_FRAME_HEIGHT' was not declared in this scope
    || m_capture->get(CV_CAP_PROP_FRAME_HEIGHT) < 480) {
                      ^~~~~~~~~~~~~~~~~~~~~~~~
/home/jose/performous-git/src/performous-git/game/webcam.cc:37:22: note: suggested alternative: 'GL_UNPACK_IMAGE_HEIGHT'
    || m_capture->get(CV_CAP_PROP_FRAME_HEIGHT) < 480) {
                      ^~~~~~~~~~~~~~~~~~~~~~~~
                      GL_UNPACK_IMAGE_HEIGHT
/home/jose/performous-git/src/performous-git/game/webcam.cc:43:22: error: 'CV_CAP_PROP_FRAME_WIDTH' was not declared in this scope
    << m_capture->get(CV_CAP_PROP_FRAME_WIDTH) << "x"
                      ^~~~~~~~~~~~~~~~~~~~~~~
/home/jose/performous-git/src/performous-git/game/webcam.cc:44:22: error: 'CV_CAP_PROP_FRAME_HEIGHT' was not declared in this scope
    << m_capture->get(CV_CAP_PROP_FRAME_HEIGHT) << std::endl;
                      ^~~~~~~~~~~~~~~~~~~~~~~~
/home/jose/performous-git/src/performous-git/game/webcam.cc:44:22: note: suggested alternative: 'GL_UNPACK_IMAGE_HEIGHT'
    << m_capture->get(CV_CAP_PROP_FRAME_HEIGHT) << std::endl;
                      ^~~~~~~~~~~~~~~~~~~~~~~~
                      GL_UNPACK_IMAGE_HEIGHT
make[2]: *** [game/CMakeFiles/performous.dir/build.make:895: game/CMakeFiles/performous.dir/webcam.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:164: game/CMakeFiles/performous.dir/all] Error 2

```

--- a/game/webcam.cc
+++ b/game/webcam.cc
@@ -33,21 +33,21 @@ Webcam::Webcam(int cam_id):
 			throw std::runtime_error("Could not initialize webcam capturing!");
 	}
 	// Try to get at least VGA resolution
-	if (m_capture->get(CV_CAP_PROP_FRAME_WIDTH) < 640
-	  || m_capture->get(CV_CAP_PROP_FRAME_HEIGHT) < 480) {
-		m_capture->set(CV_CAP_PROP_FRAME_WIDTH, 640);
-		m_capture->set(CV_CAP_PROP_FRAME_HEIGHT, 480);
+	if (m_capture->get(cv::CAP_PROP_FRAME_WIDTH) < 640
+	  || m_capture->get(cv::CAP_PROP_FRAME_HEIGHT) < 480) {
+		m_capture->set(cv::CAP_PROP_FRAME_WIDTH, 640);
+		m_capture->set(cv::CAP_PROP_FRAME_HEIGHT, 480);
 	}
 	// Print actual values
 	std::cout << "Webcam frame properties: "
-	  << m_capture->get(CV_CAP_PROP_FRAME_WIDTH) << "x"
-	  << m_capture->get(CV_CAP_PROP_FRAME_HEIGHT) << std::endl;
+	  << m_capture->get(cv::CAP_PROP_FRAME_WIDTH) << "x"
+	  << m_capture->get(cv::CAP_PROP_FRAME_HEIGHT) << std::endl;
 
 	// Initialize the video writer
 	#ifdef SAVE_WEBCAM_VIDEO
-	float fps = m_capture->get(CV_CAP_PROP_FPS);
-	int framew = m_capture->get(CV_CAP_PROP_FRAME_WIDTH);
-	int frameh = m_capture->get(CV_CAP_PROP_FRAME_HEIGHT);
+	float fps = m_capture->get(cv::CAP_PROP_FPS);
+	int framew = m_capture->get(cv::CAP_PROP_FRAME_WIDTH);
+	int frameh = m_capture->get(cv::CAP_PROP_FRAME_HEIGHT);
 	int codec = CV_FOURCC('P','I','M','1'); // MPEG-1
 	std::string out_file = (getHomeDir() / "performous-webcam_out.mpg").string();
 	m_writer.reset(new cv::VideoWriter(out_file.c_str(), codec, fps > 0 ? fps : 30.0f, cvSize(framew,frameh)));

